guide/content/en/guide/basics/request.md

```diff
@@ -167,7 +167,6 @@ The `request.ctx` object by default is a `SimpleNamespace` object allowing you t
 
 This is often used to store items like authenticated user details. We will get more into [middleware](./middleware.md) later, but here is a simple example.
 
-```python
 @app.on_request
```

Comment:
Looks like the line you have removed should be kept but that the one at line 164 (above) be deleted. The ### heading and text paragraph in between lines 164 and 170 are Markdown, not Python.